test_communication/test/test_requester_replier.py.in

```diff
@@ -25,6 +25,7 @@ def test_requester_replier():
         replier_rmw_env_key = 'RCL_ASSERT_RMW_ID_MATCHES'
     replier_env = dict(os.environ)
     replier_env[replier_rmw_env_key] = '@REPLIER_RMW@'
```

Comment:
Good catch. I updated both files in 60600784b45f189ad27c3c898aa0f942c2715ba4. Links to new CI builds coming soon on the parent ticket...